src/biogeochem/CNCStateUpdate1Mod.F90

```diff
@@ -8,7 +8,7 @@ module CNCStateUpdate1Mod
   use shr_log_mod                        , only : errMsg => shr_log_errMsg
   use clm_varpar                         , only : ndecomp_cascade_transitions, nlevdecomp
   use clm_time_manager                   , only : get_step_size_real
-  use clm_varpar                         , only : i_met_lit, i_cel_lit, i_lig_lit, i_cwd
+  use clm_varpar                         , only : i_met_lit, i_litr2, i_litr3, i_cwd
```

Comment:
I replaced `i_cel_lit` and `i_lig_lit` with `i_litr2` and `i_litr3` to allow us the flexibility of using one, two, or three litter pools regardless of their exact definition.